pkg/registry/core/service/storage/storage_test.go

```diff
@@ -561,29 +561,79 @@ func TestServiceDefaultOnRead(t *testing.T) {
 		input  runtime.Object
 		expect runtime.Object
 	}{{
-		name:   "no change v4",
-		input:  svctest.MakeService("foo", svctest.SetClusterIPs("10.0.0.1")),
-		expect: svctest.MakeService("foo", svctest.SetClusterIPs("10.0.0.1")),
+		name:  "no change v4",
+		input: svctest.MakeService("foo", svctest.SetClusterIPs("10.0.0.1")),
+		expect: svctest.MakeService("foo", svctest.SetClusterIPs("10.0.0.1"),
+			svctest.SetIPFamilyPolicy(api.IPFamilyPolicySingleStack),
+			svctest.SetIPFamilies(api.IPv4Protocol)),
+	}, {
+		name:  "missing clusterIPs v4",
+		input: svctest.MakeService("foo", svctest.SetClusterIP("10.0.0.1")),
+		expect: svctest.MakeService("foo", svctest.SetClusterIPs("10.0.0.1"),
+			svctest.SetIPFamilyPolicy(api.IPFamilyPolicySingleStack),
+			svctest.SetIPFamilies(api.IPv4Protocol)),
```

Comment:
this is a copy paste error, right? it is the same test that the below test "no change v4"

Reply:
This one sets `clusterIP` (singlular) but not plural.

Reply:
wow, so subtle, I swear I starred at the tests for 5 mins before commenting :upside_down_face: